beacon/blockchain/service.go

```diff
@@ -104,13 +104,15 @@ func (s *Service) Name() string {
 	return "blockchain"
 }
 
+// Start only starts the deposit catchup fetcher. TODO: remove.
```

Comment:
Start is part of the service.Basic interface. Does the comment mean remove the depositCatchupFetched or the whole Start function?

Reply:
Left as an exercise to the reader.

jk haha. the deposit catchup fetcher does need to be removed. Once that is removed the Start function has no purpose so maybe that can be removed as well.

Reply:
fair